src/main/java/meteordevelopment/meteorclient/systems/modules/world/HighwayBuilder.java

```diff
@@ -516,6 +517,23 @@ public MutableText getStatsText() {
         return text;
     }
 
+    private void constantAxis(Vec3d initPos) {
+
+        // Get to old y-level if offset
+        if ((int) mc.player.getY() < (int) initPos.getY()) {
```

Comment:
Is it really necessary to cast both to int?

Reply:
Not really